### PR TITLE
Back to top button added

### LINKED ISSUE #474: Include a back-to-top button on scrolling down
On scrolling the website, a back-to-top button can be included in the bottom-right corner to make it easier to navigate back to the top. Only show the back to top button when the use moves the page. Ensure you use the existing scrolling implementation.

This issue is available for anyone. You can just start working on it without getting assigned.

### PR DESCRIPTION
**Pull request by imrrahul**
In this Pull Request I have solved the issues #474  #197  ,i.e, added the back to top button as previous back to top link was not properly visible.
- Fixes #474 #197 

## Screenshot
screenshot from 2019-02-17 15-46-34

## Preview link
https://imrrahul.github.io/fosssasia.org/

### CODE REVIEW

**Comment by imrrahul** [comment]
@realslimshanky review this PR.

**Review by abhinavsagar — APPROVED** [review]

**Review by samagragupta — APPROVED** [review]

**Review by samagragupta — COMMENTED** [review]
Please resolve conflicts.

File: about/index.html [hunk 1]
```diff
@@ -762,6 +763,24 @@ <h1>Friends and Partners</h1>
...
+    document.getElementById("topbtn").style.display = "block";
+  } else {
+    document.getElementById("topbtn").style.display = "none";
+  }
+}
+
+// When the user clicks on the button,go to top of the site
+function topFunc() {
+ $("html, body").animate({ scrollTop: 0 }, "slow");
+
```
Comment (abishekvashok) [inline about/index.html:781]: Remind this empty line and move the script to script.js

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by mariobehling** [comment]
It was a good start, but such a pity you did not finish this. Closing it therefore.

**Pull request closed without merging**